Dashboard.go changed to also processed .gz files

eirikrvw opened the pull request:
Origin dashboard.go only considers .nmon files for processen. Provides code will fix this issue and also consider other files.

adejoux commented:
Hello, 

Thanks :) It's a nice solution :) I am ok to merge it. Just for you, it seems you have a invalid email address in your git configuration. it's shown in the participants section. Let me know if you want to change it before merge.

Also you should remove the line 60:

````
 fmt.Printf("%s\n\n", nmon.OS)
````

I think it's only for debugging. 


Regards,

Alain

eirikrvw commented:
Alain,

Thank you - you are right I removed debugging code and did a new commit to reflect the changes. I changed my email address so it is accepted by github. 

Regards,

Erik

adejoux commented:
Thanks. Merged :)
